packages/bindings/src/lib.rs

```diff
@@ -6,7 +6,7 @@
 extern crate core;
 #[cfg(feature = "iterators")]
 pub mod iter;
-#[cfg(all(not(target_arch = "wasm32"), test))]
+#[cfg(all(not(target_arch = "wasm32"), feature = "mocks"))]
```

Comment:
I think that the mocks feature is not needed.
If we have just #[cfg(not(target_arch = "wasm32"))] should be enough since we expose the `mocks` module when compiliting the tests for your machine.
```suggestion
#[cfg(not(target_arch = "wasm32"))]
```

Reply:
Since the mocks mod uses the `query` and `msg` features. If we don't use `mocks` feature, then it would be:
```rust
#[cfg(all(not(target_arch = "wasm32"), feature = "query", feature = "msg"))]
```

Reply:
Ok let's just keep the `mocks` feature